buildenv/jenkins/getDependency

```diff
@@ -22,7 +22,7 @@ def testBuild() {
 				sh 'tar -xzf OpenJDK8U-jdk_x64_linux_hotspot*.gz -C ${WORKSPACE}/j2sdk-image --strip-components 1'
 				sh '${WORKSPACE}/j2sdk-image/jre/bin/java -version'
 				sh 'git clone https://github.com/AdoptOpenJDK/openjdk-systemtest'
```

Comment:
Could you also update AdoptOpenJDK/openjdk-systemtest?

Reply:
Done